packages/ToggleSwitch/package-lock.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Please, don't commit this file. This file is needed locally only. In most cases this lock file should not be generated.

This usually happens when there are some inconsistencies between dependencies versions across different packages. This problem can be fixed by running `npx lernaupdate --dedupe`.